boot/zephyr/boards/nucleo_h7s3l8.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &boot_partition;
```

Comment:
this file can go, app.overlay already does this

Reply:
Oh, yes I did not know about that _app.overlay_ ; removing the board overlay then